### PR TITLE
handle newlines in comments

### LINKED ISSUE #362: Comment blocks in language files don't allow newlines
Newline '\n' is not working in comment.
image

"create_a_unordered_mutable_list": {
      "code": "let arrayName: type[];",
      "comment": "Examples: let scores : (string | number)[];\nlet nums : number[] = [12.34,1,2,3];\nlet arr : Array<any> = [1,2,3,'hi'];\nlet arr : Array<number|boolean> = [1, true, 2];"
    },

_Originally posted by @abhishek7997 in https://github.com/codethesaurus/codethesaur.us/issues/345#issuecomment-939410179_

### PR DESCRIPTION
**Pull request by INeedAUniqueUsername**
## What GitHub issue does this PR apply to?

Fixes #362

## What changed and why?

`comparecard.html` now calls `linebreaksbr` to convert `comment` newlines into `<br>` 

Before: `{{ comment|markdownify }}`
After: `{{ comment|markdownify|linebreaksbr }}`

## (If editing Django app) Please add screenshots

```
"comment": "Examples: let scores : (string | number)[];\nlet nums : number[] = [12.34,1,2,3];\nlet arr : Array = [1,2,3,'hi'];\nlet arr : Array<number|boolean> = [1, true, 2];"
```

[Before]
image

[After]
image

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [x] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

**Comment by INeedAUniqueUsername** [comment]
I'm not sure how to run tests
1. `python3 manage.py test` tells me that `markdownify.apps` is not installed
2. `pip install markdownify.apps` tells me there's no version available.

**Comment by geekygirlsarah** [comment]
> 
> 
> I'm not sure how to run tests
> 
>     1. `python3 manage.py test` tells me that `markdownify.apps` is not installed
> 
>     2. `pip install markdownify.apps` tells me there's no version available.

I've gotten this before too. Markdownify is the name of the module, but Django's looking in the apps config for it. The easiest way to solve is just 

`pip install -r requirements.txt`
(or pip3 if needed)

File: web/tests/test_templates.py [hunk 1]
```diff
@@ -74,3 +74,25 @@ def test_comparecard(self):
             "    </div>\n"
             "</div>"
         )
+
+        # no code and comment (multiple lines)
+        rendered_template_4 = render_to_string(
+            "comparecard.html",
+            {
+                "code": "",
+                "comment": "I am a humble\nmulti-line comment in the\nform of a haiku"
```
Comment (geekygirlsarah) [inline web/tests/test_templates.py:83]: AMAZING!

File: web/tests/test_templates.py [hunk 2]
```diff
@@ -74,3 +74,25 @@ def test_comparecard(self):
...
+            }
+        ).strip()
+        self.assertEquals(
+            rendered_template_3,
```
Comment (geekygirlsarah) [inline web/tests/test_templates.py:87]: There's `rendered_template_4` above but you're asserting its output to `rendered_template_3`. This is probably at least part of the failed test.

Comment (geekygirlsarah) [reply web/tests/test_templates.py:87]: image

**Comment by geekygirlsarah** [comment]
Hey @INeedAUniqueUsername, it's been a little over a week since I suggested changes. How's it going? Do you have any questions or need anything from me?

**Comment by geekygirlsarah** [comment]
Hey @INeedAUniqueUsername, how is this coming along? Do you have any questions or anything you need from me to finish this PR?

File: web/tests/test_templates.py (hunk 2 above)
Comment (geekygirlsarah) [reply web/tests/test_templates.py:87]: I pushed a change that should fix this since I never heard from you and want to close this PR.

**Comment by geekygirlsarah** [comment]
Since @INeedAUniqueUsername's original repo/branch is deleted and I merged changes into CT's `main`, this is effectively merged in but I can't actually merge it from here. @INeedAUniqueUsername's original commits are in `main` so they can still show up in contribution graphs. The only real thing I can do at this point is close this PR.

**Pull request closed without merging**